src/components/navigation-accordion/navigation-accordion.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Icon from '@mapbox/react-icon';
+import NavigationDropdown from '../navigation-dropdown/navigation-dropdown';
+
+class NavigationAccordion extends React.PureComponent {
+  render() {
+    const { props } = this;
+    const secondLevelContent =
+      props.contents.seccondLevelItems &&
+      props.contents.seccondLevelItems.map(item => {
+        return (
+          <li key={item.slug} className="mt6">
+            <a
+              href={`#${item.slug}`}
+              className="color-blue-on-hover text-decoration-none unprose"
+            >
+              {item.text}
+            </a>
+          </li>
+        );
+      });
+    const firstLevelContent = props.contents.firstLevelItems.map(
+      (page, index) => {
+        const title = page.title;
+        let icon = null;
+        const isActive = this.props.currentPath === page.path;
+        const breakLineClasses = classnames('py3 ', {
+          'border-t border--gray-light': index !== 0
+        });
+        const textClasses = classnames('pl12 py12 txt-bold txt-m flex-child', {
+          'color-black pb3': isActive
+        });
+        const activeSectionClasses = classnames(
+          'pl24-mm px0 block-mm none pr12',
+          {
+            'bg-lighten75': isActive
+          }
+        );
+        if (!isActive) {
+          icon = (
+            <Icon
+              name="chevron-down"
+              className="flex-child flex-child--no-shrink icon color-gray h24 w24"
```

Comment:
I believe it's more reliable for IE11 if you put the flexbox layout wrappers on a div outside the `<Icon>` instead of on the icon itself.